src/contractTest/java/uk/gov/hmcts/reform/em/npa/consumer/NpaPactRedactionConsumerTest.java

```diff
@@ -0,0 +1,105 @@
+package uk.gov.hmcts.reform.em.npa.consumer;
```

Comment:
Please create a separate provider class. A good approach is 1 consumer to 1 provider. 

- Allows ease of testing, 
- Debugging
- Avoids overwriting consumer when investigating locally